index.html

```diff
@@ -4545,28 +4545,172 @@ <h2>
       </p>
     </section>
 
-    <section>
-      <h2>
-Key Revocation and Recovery
-      </h2>
 
-      <p>
+
+<section>
+  <h2>
+  Verification Method Rotation
+  </h2>
+  
+  <p>
+    Verification method rotation is a proactive security measure.
+  </p>
+  
+  <p>
+    When a verification method has been active for a long time, 
+    or used for many operations, a controller might wish to perform a rotation.
+  </p>
+
+  <p class="advisement">
+    It is considered a best practice to perform verification method rotation on a regular basis.
+  </p>
+  
+  <p>
+    Section <a href="#method-operations"></a> specifies the <a>DID</a> operations to
+    be supported by a <a>DID method</a> specification, 
+    including <a href="#update">update</a>
+    which MUST be used to perform a verification method rotation.
+  </p>
+  
+  <p>
+    A controller performs a rotation when they change 
+    <code>publicKeyJwk</code> or <code>publicKeyBase58</code>, or other verification material
+    without changing the <code>id</code> of the verification method.
```

Comment:
We don't constrain an `id` to always point to an item of the same type, let along the same verification method. That is, `#key1` could be a service endpoint in one version of a DID doc, a signing key in another version, and an encryption key in another version. What you mean here, I think, is that we change *only* the value of a key, but none of its associated semantics/role/purpose.

Reply:
Yeah, I'm not a fan of this approach either -- and it's not really what I consider "rotation" either. Rather, I'd expect rotation to involve *adding* new verification methods to the DID Document without removing existing ones (and thus, their proofs continue to be valid). Internally/privately, the private key material/access to it is changed such that it is either made inactive or destroyed (but this is the private aspect of rotation, having nothing to do with the DID Document). I don't think we should encourage swapping key materials while keeping IDs the same -- I think that's a recipe for confusion.

Reply:
I agree with both your comments, perhaps I should remove this section on rotation... its telling that the word "rotation" does not appear once here: https://cheatsheetseries.owasp.org/cheatsheets/Key_Management_Cheat_Sheet.html

Reply:
However what I wrote is in line with https://developer.okta.com/docs/concepts/key-rotation/